### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by DSR1729**
Resolves Issue #194
Hey @lalitmee kindly review my PR and do let me know the changes I need to make if needed !!

### CODE REVIEW

**Comment by DSR1729** [comment]
Since my pr has passed all the checks @lalitmee can you please merge it !!

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Description 📝
+
+Please provide a brief description of the purpose and goals of this pull request.
+
+## Changes 💬
+
+- High-level overview of the changes introduced.
+- Step-by-step breakdown of the implementation, if applicable.
+
+## Checklist✅
```
Comment (lalitmee) [inline pull_request_template.md:10]: Please leave a space between emoji and text like you have added in the previous heading

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- [ ] Added appropriate test cases, if applicable.
+- [ ] Considered potential security or performance implications.
+- [ ] Code adheres to the project's style guidelines.
+- [ ] Addressed all reviewer comments and concerns, if any.
+
+## Screenshots (if applicable)
+
+Include relevant screenshots or GIFs demonstrating the changes, especially for UI-related updates.
+
+## Additional Notes📒
```
Comment (lalitmee) [inline pull_request_template.md:24]: Same here

**Review by lalitmee — CHANGES_REQUESTED** [review]
@DSR1729, can you please share a screenshot of the template? How it is looking?

File: pull_request_template.md (hunk 1 above)
Comment (DSR1729) [reply pull_request_template.md:10]: How is this ?? 👇 , any further changes you want me to make ?
Screenshot 2023-05-29 at 12-12-13 Editing GyanaGuru_pull_request_template md at pull_request_template · DSR1729_GyanaGuru · GitHub

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] Addressed all reviewer comments and concerns, if any.
+
+## Screenshots (if applicable)
+
+Include relevant screenshots or GIFs demonstrating the changes, especially for UI-related updates.
+
+## Additional Notes 📒
+
+Include any additional information or context that might be helpful for the reviewers. 
+***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***
```
Comment (lalitmee) [inline pull_request_template.md:27]: Not like this. It should be general and it should come in the `Description` section.

We can use something like this:
```
Fixes/Closes: <issue-link>
```

**Review by lalitmee — CHANGES_REQUESTED** [review]

File: pull_request_template.md (hunk 3 above)
Comment (DSR1729) [reply pull_request_template.md:27]: got it, so you are saying me to do something like this?? 👇
Screenshot 2023-05-29 at 18-08-35 Editing GyanaGuru_pull_request_template md at pull_request_template · DSR1729_GyanaGuru

Comment (lalitmee) [reply pull_request_template.md:27]: move the `Fixes/Close: <issue/link>` one line up

Comment (DSR1729) [reply pull_request_template.md:27]: Is this fine ?
Screenshot 2023-05-29 at 21-48-32 Editing GyanaGuru_pull_request_template md at pull_request_template · DSR1729_GyanaGuru · GitHub

Comment (lalitmee) [reply pull_request_template.md:27]: I meant `Fixes/Closes: <issue-link>` in one line after the description. Like this:
```markdown
## Description 📝

<!-- NOTE: Remove the one which doesn't apply to your PR. If your PR is fixing something then remove `/Closes` -->
Fixes/Closes: <issue-link>
Please provide a brief description of the purpose and goals of this pull request.
```

**Review by PiyushKalyanpy — APPROVED** [review]
Thank you @DSR1729 for your valuable contribution 🎉.We look forward to more contributions from your side.

**Pull request merged**